src/test/java/com/conveyal/gtfs/loader/JDBCTableWriterTest.java

```diff
@@ -255,6 +266,14 @@ public void canCreateUpdateAndDeleteFares() throws IOException, SQLException, In
         assertThat(updatedFareDTO.fare_id, equalTo(updatedFareId));
         assertThat(updatedFareDTO.fare_rules[0].fare_id, equalTo(updatedFareId));
 
+        // Ensure transfers value is updated correctly.
+        ResultSet updatedResult = getResultSetForId(createdFare.id, Table.FARE_ATTRIBUTES, "transfers");
+        while (updatedResult.next()) {
+            Object value = updatedResult.getObject(1);
+            LOG.info("fare#transfers {}", value);
+            assertThat(value, equalTo(0));
```

Comment:
Can you also add some other assertions for other fields to make this a more comprehensive test of whether the values are stored correctly in the database?